README.md

```diff
@@ -41,6 +43,12 @@ To complete the DevOps Academy, you will have to go through 8 classes with hands
 
 You will also be required to peer review at least 5 pull requests from other students to finish this course.
 
+There will be 2 modes of learning the content in these classes:
+
+1. Self-learning. Where you go through all the classes on your own pace and submit the exercises when you're ready.
+
+2. Face-to-Face Classes or Virtual Classes. Where the instructors will go through the content of each class and answer any questions or concerns you might have. To sign up for the classes you can email help@devopsacademy.com to get the presentation dates and the availability for the upcoming class.
+
```

Comment:
Any thought or comments about the modes of learning?

Reply:
you can add a section saying `there is no future classes yet, we are working to open a new term as soon as possible, any interest email us at help@devopsacademy.com.au` something on these lines